samples/graphene-hello/hello.manifest.template

```diff
@@ -34,7 +34,11 @@ sgx.thread_num = 16
 fs.mount.etc.type = "chroot"
 fs.mount.etc.path = "/etc"
 fs.mount.etc.uri = "file:/etc"
-sgx.trusted_files.nss = "file:$(GRAPHENEDIR)/Runtime/libnss_dns.so.2"
-sgx.trusted_files.resolv = "file:$(GRAPHENEDIR)/Runtime/libresolv.so.2"
+sgx.trusted_files.nss = "file:{{ graphene.runtimedir() }}/libnss_dns.so.2"
+sgx.trusted_files.resolv = "file:{{ graphene.runtimedir() }}/libresolv.so.2"
 sgx.allowed_files.hosts = "file:/etc/hosts"
 sgx.allowed_files.resolv = "file:/etc/resolv.conf"
+
+
+# Name Service Switch (NSS) files, see 'man nsswitch.conf'
+sgx.allowed_files.nsswitch  = "file:/etc/nsswitch.conf"
```

Comment:
```suggestion
sgx.allowed_files.nsswitch  = "file:/etc/nsswitch.conf"
```
Do we need nsswitch at all?

Reply:
I added this, because the application tried to access it, complained that it had no access and paniced. Was fixed by adding this line. Maybe there is a way, so that the application doesn't try to access it?

Reply:
It's ok to add if it's needed. Just wondering why it's needed now and wasn't before.